<?xml version="1.0" ?>
<launch>

    <!-- Aruco Detector -->
    <node name="aruco_tracker" pkg="aruco_ros" type="single">
        <remap from="/camera_info"          to="/camera/rgb/camera_info" />
        <remap from="/image"                to="/camera/rgb/image_raw" />
        <param name="image_is_rectified"    value="true"/>
        <param name="marker_size"           value="0.088"/>
        <param name="marker_id"             value="300"/>
        <param name="reference_frame"       value="camera_link"/>
        <param name="camera_frame"          value="camera_rgb_optical_frame"/>
        <param name="marker_frame"          value="camera_marker" />
    </node>

    <!-- Calibration node -->
    <include file="$(find easy_handeye)/launch/calibrate.launch">
        <arg name="eye_on_hand"             value="false"/>
        <arg name="start_sampling_gui"      value="false"/>

        <arg name="robot_base_frame"        value="base_link"/>
        <arg name="robot_effector_frame"    value="flange"/>

        <arg name="tracking_base_frame"     value="camera_link"/>
        <arg name="tracking_marker_frame"   value="camera_marker"/>
        
        <arg name="freehand_robot_movement" value="true"/>
        <arg name="start_rviz"              value="false"/>
    </include>

</launch>